packages/amazonq/.changes/next-release/Bug Fix-dd58d9cb-4fb4-4cb3-a9a2-dc5c5462e529.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q (/dev): Allow end-user access history diff generated along iterations"
```

is the customer going to know what this means? is there a way to express this so that it's more meaningful to the customer? See [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).

I don't know if this is correct, but something like:

```suggestion
	"description": "Amazon Q (/dev): view diffs of previous /dev iterations"
```

---

SGTM